src/main/java/uk/gov/hmcts/reform/sscs/domain/wrapper/SyaCaseWrapper.java

```diff
@@ -136,15 +136,20 @@ public void setSignAndSubmit(SyaSignAndSubmit signAndSubmit) {
     }
 
     public SyaContactDetails getContactDetails() {
+        if (null == appellant) {
```

Comment:
TODO: Add junit test to cover different scenarios such as: only appellant is null, only appointee is null, both are null and both are populated

Reply:
Was this done?

Reply:
it's not, but it's not a blocker for now and I will address as part of the next ticket if that's ok.

Reply:
ok